src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -130,18 +119,62 @@ const blastApiSlice = restApiSlice.injectEndpoints({
         }
       }
     }),
-    fetchBlastSubmission: builder.query<BlastJobResultResponse, string>({
-      query: (jobId) => ({
-        url: `${config.toolsApiBaseUrl}/blast/jobs/result/${jobId}/json`
-      })
+    fetchAllBlastRawResults: builder.query<{ result: string }[], string[]>({
+      queryFn: async (jobIds, _queryApi, __extraOptions, baseQuery) => {
```

Comment:
Out of curiosity, what is the difference between using `_` &  `__`?

Reply:
✅